internal/functions/Invoke-ManagedComputerCommand.ps1

```diff
@@ -33,7 +33,7 @@ function Invoke-ManagedComputerCommand {
         [PSCredential]$Credential,
         [Parameter(Mandatory)]
         [scriptblock]$ScriptBlock,
-        [string[]]$ArgumentList,
+        [object[]]$ArgumentList,
```

Comment:
This should not affect other commands that use this function but would not hurt to verify a few that are not included in our test framework.

Reply:
Ok, I'll have a look. These commands use Invoke-ManagedComputerCommand with -ArgumentList:
Enable-DbaAgHadr
Get-DbaStartupParameter
Set-DbaStartupParameter
Get-DbaTcpPort
Set-DbaTcpPort
Test-DbaSpn
Update-DbaServiceAccount
Get-WmiHadr